packages/core/src/browser/shell.ts

```diff
@@ -26,6 +26,7 @@ import {
 import {
     VirtualElement, h
 } from '@phosphor/virtualdom';
+import { StatusbarImpl, StatusbarLayoutData } from "./statusbar/statusbar";
```

Comment:
an interface is not the widget

Reply:
I think I find this confusing since:

There's 

StatusBar Interface: Which is an interface to the StatusBar model
StatusBarImpl: Which is a widget that implements that model.

So reading StatusBarImpl I'm expecting a model , not a widget.

Seems like having:

StatusBarWidget 
    - model : StatusBarImpl
   
And the import StatusBarWidget would make more sense ?
It would also make the StatusBarImpl testable, especially given that more functions from StatusBarEntry service would be merged into it ?